hasjob/models/campaign.py

```diff
@@ -383,6 +383,9 @@ class CampaignView(TimestampMixin, db.Model):
     Track users who've viewed a campaign
     """
     __tablename__ = 'campaign_view'
+    #: datetime to log db activity
```

Comment:
Change this comment in all models to: "Datetime when this activity happened (which is likely much before it was written to the database)"